tests/system/projects/test_project.py

```diff
@@ -1185,3 +1185,24 @@ def test_load_project_remotely_with_secrets_failed(self):
         assert bg_task.status.state == mlrun.common.schemas.BackgroundTaskState.failed
         with pytest.raises(mlrun.errors.MLRunNotFoundError):
             db.get_project(name)
+
+    def test_remote_workflow_schedule_task_name(self):
+        # Test running remote workflow when the project files are store in a relative path (the subpath)
```

Comment:
this is a heavy test for a simple change, I would not overload our system test for that. you may use a simple unit test which would be much faster and covered by our regular CI